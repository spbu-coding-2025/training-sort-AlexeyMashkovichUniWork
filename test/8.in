790919343 -64359526 1014462632 -4056187 -548712788 -71286059 -354309188 1060570063 -275980175 792128139 635638587 -1073579063 699688731 778883447 -233781859 957811913 774805050 -925963338 -544638729 -417577647 -740620948 586711954 -198994547 893839438 745921156 616576122 -14910779 -608024612 239788809 236552690 -131282892 -43033672 -901548661 -190562085 1026651964 -376519626 811893679 -401399049 -389691388 -537828321 -683012735 -827794626 -537665561 -1057065829 1024830644 302294403 974487907 725893869 450072888 -643892647 -765425603 783193763 1016561130 109321674 603291376 688740462 -347844029 -485361227 -993025975 965686604 824933286 -50567043 -151088893 997126448 832612695 -198178754 -453135003 570764550 474164020 230915432 -1040805596 864893108 476862630 -504729333 881569102 427951449 871306893 782315185 80103494 247637957 -935319287 388419714 -42910105 -992499981 -576000437 -513360553 769982304 149897358 75020043 850698152 41842137 -173788496 -273610716
