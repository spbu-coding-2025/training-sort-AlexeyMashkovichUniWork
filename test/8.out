-1073579063 -1057065829 -1040805596 -993025975 -992499981 -935319287 -925963338 -901548661 -827794626 -765425603 -740620948 -683012735 -643892647 -608024612 -576000437 -548712788 -544638729 -537828321 -537665561 -513360553 -504729333 -485361227 -453135003 -417577647 -401399049 -389691388 -376519626 -354309188 -347844029 -275980175 -273610716 -233781859 -198994547 -198178754 -190562085 -173788496 -151088893 -131282892 -71286059 -64359526 -50567043 -43033672 -42910105 -14910779 -4056187 41842137 75020043 80103494 109321674 149897358 230915432 236552690 239788809 247637957 302294403 388419714 427951449 450072888 474164020 476862630 570764550 586711954 603291376 616576122 635638587 688740462 699688731 725893869 745921156 769982304 774805050 778883447 782315185 783193763 790919343 792128139 811893679 824933286 832612695 850698152 864893108 871306893 881569102 893839438 957811913 965686604 974487907 997126448 1014462632 1016561130 1024830644 1026651964 1060570063
